modules/debug.js

```diff
@@ -1,31 +1,60 @@
+var KeyCode = {
+  DEBUG : 68,
+  PAUSE : 80
+}
 var Debug = function()
 {
-  this.debugMode = true;
+  this.population = null;
+  this.debugMode = false;
   this.statsInterval = null;
+  this.paused = false;
 
   var self = this;
 
-  $("html").click(function()
+  $(document).bind("keyup", function(e)
   {
-    self.debugMode = !self.debugMode;
-
-    if(self.debugMode)
+    if(e.shiftKey && e.keyCode == KeyCode.DEBUG)
     {
-      self.Start();
+      self.debugMode = !self.debugMode;
+
+      if(self.debugMode)
+      {
+        self.Start();
```

Comment:
maybe we should use shorthand if for this one because it's a simple if/else case 
(self.debugMode) ? self.Start() : self.Stop()